fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfReporterStep.java

```diff
@@ -281,6 +281,9 @@ public static String generate(CadfReporterStep obj)
 
         public static void generate(CadfReporterStep obj, JsonGenerator generator)
```

Comment:
maybe some javadoc to explain the behavior when null is passed?

Reply:
added comment